lib/permutations.rb

```diff
@@ -1,4 +1,16 @@
 
 def permutations?(string1, string2)
```

Comment:
This works, but it's O(n^2) which isn't great.  Think about how a hash could be used to make this faster.

Reply:
I believe I fixed this. What is the proper way to send it back to you, if needed?